vmdb/app/models/repository.rb

```diff
@@ -59,17 +59,17 @@ def path=(newpath)
 
   def vms
     if relative_path != "/"
-      Vm.all(:conditions=>["storage_id=? and location like ?", storage_id, relative_path+"%"])
+      Vm.where("storage_id=? and location like ?", storage_id, (relative_path + "%")).to_a
```

Comment:
Useless parens (unless you think it makes it clearer, but in that case we should just interpolate.)